src/schema/clan/view.ts

```diff
@@ -0,0 +1,20 @@
+import { Type } from "@sinclair/typebox";
+
+import { defineEndpoint } from "@/generator-helpers.js";
+
+export default defineEndpoint({
+    source: "user",
+    target: "server",
+    description: "Get information about a clan.",
+    request: {
+        data: Type.Object({
+            clanId: Type.Ref("clanId"),
```

Comment:
Is the pattern of passing `somethingId` on a request already scoped by `something` present in other tachyon requests?

Reply:
> Is the pattern of passing `somethingId` on a request already scoped by `something`

In the diff above we can clearly see that we are passing a data structure containing a parameter `clanId` inside a request already scoped by `clan`, in this case: `clan/view`.

> present in other tachyon requests?

Is this a pattern already employed in other requests?

---

It seems the answer is no, as [exemplified in this request](https://github.com/beyond-all-reason/tachyon/blob/master/src/schema/lobby/join.ts#L12) although the pattern might be inconsistent.

Hence the answer to this question hopefully would be no and the recommendation to be to not use `clanId`, rather `id`.

I hope I was able to clarify. If not please let me know.

Reply:
We have both naming patterns already.

We should unify on one, I don't have preferences on which one.

Reply:
That's great, thank you!